automatic/eid-belgium/tools/VERIFICATION.txt

```diff
@@ -0,0 +1,23 @@
+VERIFICATION
```

Comment:
please move this file to a `legal` directory instead, to match the convention used for most of the existing packages.

Reply:
Taken care off in https://github.com/wget/chocolatey-package-eid-belgium